### PR TITLE
Implement molo.profile

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
This is how the new user profile looks like on smart phones so far. 
profile

**Comment by smarzban** [comment]
This is how the new user profile looks like on feature phones so far.
profile-feature

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 molo.core==2.5.0
+molo.profiles<1.0.0,>=0.1.4
 molo.commenting<1.0.0,>=0.2.5
 molo.profiles>=0.2.2
```
Comment (smarzban) [inline requirements.txt]: @miltontony do we need both molo.profiles? was this intentional?

File: ndohyep/polls/results.html [hunk 2]
```diff
@@ -22,7 +22,7 @@ <h6 class="speak">{% trans "SPEAK OUT" %}</h6>
       </div>
 		</div>
 	{% endfor %}
- 	<p class="back"><a href="{% url 'vote' question.id %}">{% trans "Back" %}</a></p>
+ 	<p class="back">< <a href="/">{% trans "Back" %}</a></p>
```
Comment (miltontony) [inline ndohyep/polls/results.html]: @saeedmb8 The URL for home should be `{{request.site.root_page.url}}`

File: ndohyep/templates/profiles/done.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+{% block content %}
+<div class="profile">
+  <h2>{% trans "Thank you for joining!" %}</h2>
+  <h4>{% trans "Let us know your birthday to get access to exclusive content." %}</h4><br>
+  <form method="post" action="{% url 'molo.profiles:registration_done' %}">{% csrf_token %}
+      {{ form.date_of_birth }}
+      <br>
+      <br>
+      <input type="submit" value='{% trans "UPDATE" %}' />
+      <a href="/" class="button">{% trans "Skip" %}</a>
```
Comment (miltontony) [inline ndohyep/templates/profiles/done.html]: Same `{{request.site.root_page.url}}`

File: ndohyep/views.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+from molo.core.models import ArticlePage
+from wagtail.wagtailsearch.models import Query
+
+import django_comments
+
+from molo.commenting.forms import MoloCommentForm
+from molo.commenting.models import MoloComment
+
+
+class CommentReplyForm(TemplateView):
```
Comment (miltontony) [inline ndohyep/views.py]: Hmm...this should actually be called `CommentReplyFormView` - otherwise it can be confused with a Form

**Comment by smarzban** [comment]
@miltontony Ready for you Sir \0/

File: ndohyep/views.py (hunk 4 above)
Comment (miltontony) [inline ndohyep/views.py]: @saeedmb8 did you see this comment?

**Comment by miltontony** [comment]
:+1:  :cake: :star2: finally landing this baby!!

**Comment by smn** [comment]
:boss:

**Pull request merged**